vajram/vajram-codegen/src/main/java/com/flipkart/krystal/vajram/codegen/models/ParsedVajramData.java

```diff
@@ -51,10 +53,33 @@ public static Optional<ParsedVajramData> fromVajram(VajramInfo vajramInfo, Utils
             vajramInfo.responseType()));
   }
 
+  public static void handleDuplicateDependencyBySameResolverException(List<ExecutableElement> methods, Utils util){
```

Comment:
method name: `validateNoDuplicateResolvers`

Reply:
Change method param to `staticMethods`

Reply:
done